backend/src/main/java/io/debezium/configserver/rest/ConnectorResource.java

```diff
@@ -447,4 +447,49 @@ public Response deleteConnector(
         }
         return deleteResponse;
     }
+
+    @Path(ConnectorURIs.CONNECTOR_CONFIG_ENDPOINT)
```

Comment:
This method is just a proxy method for Kafka Connect and should therefore go to `KafkaConnectResource` class.